packages/runtime/container-runtime/src/dataStoreContexts.ts

```diff
@@ -67,6 +67,11 @@ import { FluidDataStoreContext, LocalFluidDataStoreContext } from "./dataStoreCo
         return this._contexts.get(id);
     }
 
+    public delete(id: string): boolean {
+        this.deferredContexts.delete(id);
```

Comment:
Should we delete from `notBoundContexts`?

Reply:
I am not sure actually. If a context is in `notBoundContexts`, it won't be part of the summary so we won't be attempting to delete it anyway. However, just from this DataStoreContexts perspective, it might make sense to delete it from everywhere.

Reply:
Right that's sort of what I was thinking.